internal/builders/generic/generate.go

```diff
@@ -15,14 +15,10 @@
 package main
```

Comment:
Probably we don't even need the `generate` command on the generic builder anymore? I think it was only used by tho container workflow anyway.